package.json

```diff
@@ -38,9 +38,8 @@
       "packages/**/tests/*.js",
       "packages/**/tests/**/tests.js"
     ],
-    "compileEnhancements": false,
-    "babel": false,
-    "verbose": true
+    "verbose": true,
+    "timeout": "60s"
```

Comment:
AVA now has a default timeout set of `10s` (previously it had none). See - https://github.com/avajs/ava/commit/f2172526ae03490a548f6ef30a81c8212c280c58#diff-359d7f196adb7d5fcf884234506213692106ed86f073ed3533d718e4c89b0375 and https://github.com/avajs/ava/releases/tag/v3.0.0.

For now this is _kind of_  a wild guess. But we can fine tune it as we go 👍